x/margin/keeper/open_long_test.go

```diff
@@ -64,6 +64,8 @@ func TestOpenLong_PoolDisabled(t *testing.T) {
 		msg = &types.MsgOpen{
 			Leverage:         math.LegacyNewDec(10),
 			CollateralAmount: math.NewInt(1),
+			CollateralAsset:  "aaa",
+			BorrowAsset:      "bbb",
```

Comment:
Could you use meaningful assets lik "ATOM" or "USDC" here? "aaa" and "bbb" seems not good even it's a test case.

Reply:
I do for the following test cases